src/workloads/scale/LargeScaleModel.yml

```diff
@@ -0,0 +1,469 @@
+SchemaVersion: 2018-07-01
+Owner: Storage Engines
```

Comment:
Using the new "auto-run" functionality you can have this be scheduled as part of the sys-perf waterfall by adding something like this:

```yaml
AutoRun:
  Requires:
    bootstrap:
      mongodb_setup: single-replica-15gbwtcache
```

Where `single-replica-15gbwtcache` corresponds to which mongodb_setup is required.

Reply:
That sounds very nice.  But in its current state, it will probably fail because of genny OOM, or timeout.  I'll defer that change for a later iteration.